### PR TITLE
Ctctraders 1310: Implemented Add Extra Document Information page

### PR DESCRIPTION
**Pull request by kevincoatsworth**

### CODE REVIEW

File: app/pages/addItems/AddExtraDocumentInformationPage.scala [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+package pages.addItems
+
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object AddExtraDocumentInformationPage extends QuestionPage[Boolean] {
+
+  override def path: JsPath = JsPath \ toString
```
Comment (ashwinirudrappa) [inline app/pages/addItems/AddExtraDocumentInformationPage.scala:24]: @kevincoatsworth AddExtraDocumentInformationPage should take itemIndex and documentIndex as params

File: conf/views/addItems/addExtraDocumentInformation.njk [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      <div class="govuk-grid-column-two-thirds">
+
+        {% if form.errors.length > 0 %}
+          {{ govukErrorSummary({
+            "titleText": messages("error.summary.title"),
+            "errorList": form.errors
+          }) }}
+        {% endif %}
+
+        <form action="{{ routes.controllers.addItems.AddExtraDocumentInformationController.onSubmit(lrn, mode).url }}" method="post">
```
Comment (ashwinirudrappa) [inline conf/views/addItems/addExtraDocumentInformation.njk:30]: @kevincoatsworth need to pass indexes to onSubmit method

**Review by ashwinirudrappa — APPROVED** [review]
@kevincoatsworth Code looks good  please address those  minor comments

File: app/pages/addItems/AddExtraDocumentInformationPage.scala (hunk 1 above)
Comment (kevincoatsworth) [reply app/pages/addItems/AddExtraDocumentInformationPage.scala:25]: :+1:

File: conf/views/addItems/addExtraDocumentInformation.njk (hunk 2 above)
Comment (kevincoatsworth) [reply conf/views/addItems/addExtraDocumentInformation.njk]: :+1:

**Pull request merged**